### PR TITLE
:rocket: New Contact Confirmation page Layout

### LINKED ISSUE #3500: update contact confirmation screen
context
---
We want to use the confirmation screen to add valuable resources to the confirmation screen

mockup
---
view [mockup](https://www.figma.com/file/K0VV8lc3pmlMVModYM70nj/website?node-id=9425%3A23731&t=Iw0rF69aIFEtYBYm-1)
Bildschirm­foto 2023-03-08 um 09 31 55

### PR DESCRIPTION
**Pull request by Siddhant-K-code**
## Description

:rocket: New Contact Confirmation page Layout

## Related Issue(s)

Fixes #3500

## How to test

- Try to submit [`/contact/support`](https://deploy-preview-3514--gitpod-kumquat.netlify.app/contact/support) or [`/contact/sales`](https://deploy-preview-3514--gitpod-kumquat.netlify.app/contact/sales) & see :eyes:

## Release Notes

```release-note
NONE
```

### CODE REVIEW

File: src/routes/contact/sales.svelte [hunk 1]
```diff
@@ -199,35 +199,36 @@
...
+    description: "We'll help you find the best plan for your business.",
     title: "Talk to an expert - Book a demo",
   }}
 />
 
-<div>
+{#if isEmailSent}
+  <Header
+    title="Thanks for contacting us"
+    text="we'll get back to you shortly."
```
Comment (chrifro) [inline src/routes/contact/sales.svelte:210]: Should be spelt with a capital "W"

File: src/lib/contents/support.ts [hunk 2]
```diff
@@ -68,6 +68,28 @@ export const cards: Card[] = [
...
+      href: "/customers",
+      text: "View customer stories",
+      btnVariant: "primary",
+    },
+  },
+  {
+    title: "CDEs the future of development",
+    text: "<p class='text-base'>Our whitepaper explores a business perspective on CDEs and their benefits for engineering teams.</p>",
+    link: {
+      href: "/cde",
```
Comment (chrifro) [inline src/lib/contents/support.ts:85]: linking to https://www.gitpod.io/whitepaper/cde would be the expected path

**Review by chrifro — COMMENTED** [review]
- [ ] Can you automatically scroll to the top of the page? The footer is currently in my viewport and I need to scroll up to see any context. That's very confusing

Otherwise it's great ✨

**Comment by Siddhant-K-code** [comment]
Done 😄 

https://user-images.githubusercontent.com/55068936/225076258-d3bdd218-ef92-4dcd-8224-09db16ed972d.mov

**Review by selfcontained — APPROVED** [review]
LGTM - looks great on mobile as well, nice job making it responsive.

**Pull request merged**